airbyte-webapp/src/components/EntityTable/components/StatusCellControl.module.scss

```diff
@@ -0,0 +1,5 @@
+.inProgressLabel {
+  height: 32px; /** Needs to match our Button height to avoid jumping around */
```

Comment:
Should we make this a variable? I've also had another occasion where I needed the button height

Reply:
The reason I didn't is that IMO we should not really need to pass around the height of the button. The table row should have an appropriate (larger) height so that it's not reflowing depending on the content. But of course that's a little out of scope. On second thought, a variable is at least better than a comment like this, so I'll add the button heights to `_variables.scss`